utils/Button.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import "@reach/dialog/styles.css"
+import { Dialog } from "@reach/dialog"
+
+interface ButtonProps {
+  text: string
+  type?: string
+  onClick: (params: any) => any
+}
+
+const Button = ({ onClick, text, type }: ButtonProps) => {
```

Comment:
Why is there a `button` utility? I don't see much benefit to this component. You can easily write a `<button type="submit" />` in your form 🤔